src/test/globals.js

```diff
@@ -1,3 +1,4 @@
 // @flow
-// eslint-disable-next-line no-underscore-dangle
+/* eslint-disable no-underscore-dangle */
 global.__SERVER__ = typeof document === 'undefined'
+global.__VERSION__ = '0.0.0'
```

Comment:
Can we maybe set this to sth that makes it clear that Jest is using a “mocked” global here? Possibly “JEST_MOCK_VERSION” or sth?

Reply:
Sure, makes sense